pkg/operator/operator.go

```diff
@@ -365,6 +370,10 @@ func (o *Operator) DeployApp(appID string, sequence int64) (deployed bool, deplo
 		return false, errors.Wrap(err, "failed to deploy app")
 	}
 
+	if err := o.ensureKotsadmApplicationMetadataConfigMap(util.PodNamespace, kotsKinds, app.UpstreamURI); err != nil {
```

Comment:
since we should show branding for the version even if deploying the app fails, how about we move this to the defer statement at the top and execute this regardless of deployment status?